.github/ISSUE_TEMPLATE.md

```diff
@@ -1,24 +1,25 @@
 <!-- 
-Did you spot a typo or do you want to contribute to documentation? Instead of a GitHub issue, we encourage you to submit a pull-request (https://help.github.com/articles/creating-a-pull-request/) and we will review your changes!
+Thank you for your interest in contributing!
 
-For all other issues, please fill in the sections below.
+Before submitting an issue, check our existing issues and pull requests to see if someone else has reported your issue previously. 
 
-Check existing issues and pull-requests before submitting new ones, as a courtesy to the maintainers and making sure work isn't duplicated.
+If you know how to resolve the issue you're reporting, we encourage you to submit a pull request instead: https://help.github.com/articles/creating-a-pull-request/.
+
+If you have a troubleshooting issue, reach out to our community: https://github.com/Kong/kong/discussions.
```

Comment:
Not sure how to phrase this, but I think this sentence needs to be a bit clearer - what exactly is a "troubleshooting issue"? Maybe make a point of drawing a contrast between code/product/etc troubleshooting vs something wrong with docs. 

E.g.: "If you need help troubleshooting issues with (your implementation? code? I don't know what phrase/word to use here), reach out to our community"